### PR TITLE
Broken Link and Autodate on Footer

### PR DESCRIPTION
**Pull request by aaronlippold**
- fixed the broken links by correcting the package.json and updated the component to correctly build the uri
- updated the footer component to keep the Copywrite year dynamically

Signed-off-by: Aaron Lippold <lippold@gmail.com>

### CODE REVIEW

File: src/components/global/Footer.vue [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <template>
   <v-footer app title="footer" class="bar font-weight-light">
-    <span class="bar-visible--text">The MITRE Corporation &copy; 2019</span>
+    <span class="bar-visible--text"
+      >The MITRE Corporation &copy; 2018-{{ new Date().getFullYear() }}</span
+    >
```
Comment (rbclark) [inline src/components/global/Footer.vue]: I'd really prefer avoiding this style if possible.

```javascript
    <span class="bar-visible--text">
        The MITRE Corporation &copy; 2018-{{ new Date().getFullYear() }}
    </span>
```

**Review by rbclark — CHANGES_REQUESTED** [review]
Should the Docs go somewhere different than the Github link?

File: src/components/global/Footer.vue (hunk 1 above)
Comment (aaronlippold) [reply src/components/global/Footer.vue]: which style do we prefer?

Comment (aaronlippold) [reply src/components/global/Footer.vue]: ok, updated to use a getter method

**Review by rbclark — APPROVED** [review]

**Pull request merged**